### PR TITLE
Added Say What? Shiok Food Card Game

### PR DESCRIPTION
**Pull request by xinyitay**
Submitting this PR means you have:

- [x] searched the [readme][readme-url] file and searched the [open pull requests](https://github.com/edm00se/awesome-board-games/pulls) (no duplicates please)
- [x] and will follow [the desired format](https://github.com/edm00se/awesome-board-games/blob/main/formatting.md)
- [x] used image links that resolve correctly, are appropriately sized, and are served over HTTPS

Once your PR is created, it will trigger a build a GitHub Action (Awesome-Lint). This includes a format linting process, so if there are any issues with the build, please address them.

Do you have further questions? Check the [contribution guidelines](https://github.com/edm00se/awesome-board-games/blob/main/contributing.md) or [open an issue](https://github.com/edm00se/awesome-board-games/issues/new).

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

[readme-url]: https://github.com/edm00se/awesome-board-games/edit/main/readme.md

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1080,7 +1080,15 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
 | ------: | -------: | -----: |
 |   2  |        8+ | 30m around|
 
+### [Say What? Shiok Food Card Game](https://www.saywhatwithfriends.com/shop/say-what-singapores-shiok-food-card-game)
 
+> Love Singapore’s Local Food? This Food Card Game contains 50 local dishes for you to guess and the winner will be the one who can guess the most.
+
+![Say What? Shiok Food Card Game](https://images.squarespace-cdn.com/content/v1/5649b2dbe4b0142472e867bc/1573954498990-B1NESNC79U51TJNXQEBB/SG+FOOD+GAME+FRONT.JPG?format=1000w)
```
Comment (edm00se) [inline readme.md:1087]: This image is far too large, [as is visible in the PR preview build](https://awesome-board-games-git-fork-xinyitay-adding-shi-dc53b1-edm00se.vercel.app/#say-what-shiok-food-card-game).

Please follow [the formatting guide](https://github.com/edm00se/awesome-board-games/blob/main/formatting.md) and make use of an appropriately sized image.

**Review by edm00se — CHANGES_REQUESTED** [review]
Please update this PR with a more appropriately sized image. Otherwise this PR looks good.

**Comment by xinyitay** [comment]
HI! Thanks for the comments. I tried to search online but am unable to find an image that fits the specifications. Is it possible if I upload one that I have manually resized? Thank you!

File: readme.md (hunk 1 above)
Comment (edm00se) [reply readme.md:1087]:
```suggestion
![Say What? Shiok Food Card Game](https://images.squarespace-cdn.com/content/v1/5649b2dbe4b0142472e867bc/1573954498990-B1NESNC79U51TJNXQEBB/SG+FOOD+GAME+FRONT.JPG?format=250w)

I found that with the format parameter that the image link can return a more reasonable image size. If you accept this code suggestion that should meet the needs of this PR.
```

**Review by edm00se — APPROVED** [review]

**Comment by edm00se** [comment]
> HI! Thanks for the comments. I tried to search online but am unable to find an image that fits the specifications. Is it possible if I upload one that I have manually resized? Thank you!

One of the cornerstones of this project, as I see it, is that it's meant to be purely referential in nature. As such I've avoided hosting the images here quite heavily. As it is, I found the image link you used can be adjusted for the formatter to return a different sized image. I'll commit that and call it good for now.

**Review by edm00se — APPROVED** [review]
Thanks for contributing!

**Comment by edm00se** [comment]
refs #176

**Pull request merged**